Fixed #32220 -- Added durable argument to transaction.atomic(). (#13708)
django/django: docs/releases/3.2.txt

* Added the :class:`~django.db.models.functions.Random` database function.

* The new ``durable`` argument for :func:`~django.db.transaction.atomic()`

>   guarantees the changes made in the atomic block with be committed if the

Reviewer: ```suggestion
  guarantees the changes made in the atomic block will be committed if the
```

>   block exits without error. An inner atomic block marked as durable will raise

Reviewer: ```suggestion
  block exits without error. A nested atomic block marked as durable will raise
```
